tendrl/gluster_integration/sds_sync/brick_utilization.py

```diff
@@ -59,18 +59,30 @@ def get_lvs():
             {"message": str(err)}
         )
         return None
-    out = out.split('\n')
```

Comment:
out is u'' when no lvm in storage node

Reply:
So all we are doing here is checking non empty for `out` and adding a exception handling for parsing the `out` variable.
Looks good to me. Make sure this time we test all the possible scenarios.

Reply:
ok